.github/workflows/ci.yml

```diff
@@ -646,7 +646,7 @@ jobs:
       - name: Require Symfony components
         run: composer require symfony/uid --dev --no-interaction --no-progress --ansi
       - name: Install Elasticsearch-php # there is a version matrix for this package: https://packagist.org/packages/elasticsearch/elasticsearch
-        run: composer require "elasticsearch/elasticsearch:v7.14.0" --no-interaction --no-progress --ansi
+        run: composer require "elasticsearch/elasticsearch:v7.14.0" --no-interaction --no-progress --ansi --with-all-dependencies
```

Comment:
To do in 2.6 IIUC.

Reply:
The elasticsearch/elasticsearch version supported on 2.6 is not the same and does not conflict with psr/log on 2.6 branch. This issue only occures on the main branch